platform/ios/platform/ios/MapLibre.playground/Contents.swift

```diff
@@ -145,9 +140,18 @@ responder.mapView = mapView
 let tapGesture = UILongPressGestureRecognizer(target: mapDelegate, action: #selector(mapDelegate.handleTap))
 mapView.addGestureRecognizer(tapGesture)
 
+//: Set Style
+let styleJSON = "https://raw.githubusercontent.com/roblabs/openmaptiles-ios-demo/master/OSM2VectorTiles/styles/geography-class.GitHub.json"
```

Comment:
Is it possible to move this styleJSON to this repo?

Reply:
We are working on MapLibre website which will host zero-configuration style and vector tiles. It should be ready in 2-4 weeks. I would suggest to use the style from there when it is ready.

Reply:
I'm already introducing hard coded styles, which will need to be unraveled.  

In a separate branch specific to _work on styles_, we'll need to do some maintenance & populating of style URLs in the class [MGLStyle](https://docs.mapbox.com/archive/ios/maps/api/5.9.0/Classes/MGLStyle.html).  Then in the future we can have custom MapLibre styles usable as

```swift
let style  = MGLStyle.geographyClassURL
// or
let style2 = MGLStyle.zeroConfURL
```

Reply:
> *Is it possible to move this styleJSON to this repo?*

Good question.  By chance, the style and tile data this points to are `.pbf`/protobuf files, already served up by GitHub.com.  If we were to commit tile data, it may be better to go the route of having `.mbtiles` from OpenMapTile.org